styles/styles.css

```diff
@@ -856,3 +856,23 @@ body.no-scroll {
 .section[class*='bg-primary-blue'] .main-heading p {
   color: var(--color-white);
 }
+
+
+#skip-to-main-content{
+  position: absolute !important;
```

Comment:
Why `!important`? Is something overriding this?

Reply:
Done